modules/caddytls/connpolicy.go

```diff
@@ -481,4 +515,10 @@ type ConnectionMatcher interface {
 	Match(*tls.ClientHelloInfo) bool
 }
 
+// ClientCertValidator is a type which validates client certificates.
+// It is called during verifyPeerCertificate in the TLS handshake.
+type ClientCertValidator interface {
```

Comment:
```suggestion
type ClientCertificateValidator interface {
```

Just wondering what your thoughts are on spelling out the full word here. I feel like that might be more important for the _interface_ being implemented, but I normally don't mind `Cert`, and maybe I even use it elsewhere. Not a big deal, but just thought I should suggest it.